pages/linux/gnome-screenshot.md

```diff
@@ -0,0 +1,36 @@
+# gnome-screenshot
+
+> Capture the screen, a window, or a user-defined area and save the image to a file.
+> More information: <https://apps.gnome.org/app/org.gnome.Screenshot/>.
+
+- Take a screenshot and save it to the default location, normally `~/Pictures`:
+
+`gnome-screenshot`
+
+- Take a screenshot and save it to the named file location:
+
+`gnome-screenshot --file {{path/to/file}}`
+
+- Take a screenshot and save it to the clipboard:
+
+`gnome-screenshot --clipboard`
+
+- Take a screenshot after the specified number of seconds:
+
+`gnome-screenshot --delay {{5}}`
+
+- Launch the GNOME Screenshot GUI:
+
+`gnome-screenshot --interactive`
+
+- Display the version:
+
+`gnome-screenshot --version`
```

Comment:
Honestly, I just added them since the page I copy-and-pasted to create this one had it.
Happy to remove it, I'll just wait for another opinion.

As soon as we get confirmation, I'll either move it to the bottom or remove it entirely.